### ISSUE TITLE
Accessibility - iOS - On "Legal Standards" Screen VoiceOver Not Reading Special Character Footnote

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#101]
While testing with VoiceOver, we noticed on the "Legal Standards" screen VoiceOver is not reading the special character (asterisk) as a footnote which it did previously. The footnote is at the bottom of the page, and I think was being added to the specific line as an accessible label or something like that.

**Comment by jfc3-dol** [comment]
VoiceOver is still not reading the asterisk at all let alone as the text is represents in the footer as "No conscription".

Also, I'm not sure why there an open and close bracket "( )" after the N/A* for the "State Compulsory Military Recruitment" which you can see for "India".

**Comment by harpreetsdev** [comment]
@jfc3-dol 

The voiceover on **device - iPhone 11 pro** as well **simulator** are showing no issues with reading the asterisk * statement in Legal Standards section. I haven't been able to re-create the above mentioned issues.

I have created 2 screen recording videos from my iPhone and saved them in my google drive, please find links below - 

1) **Legal-Standards-Footnote.mov** - In this video I took 2 random countries in Voice over mode and navigated to the bottom of Legal Standards section and you can see the Voice over focus goes to the Footnote text and reads it. Although you can only see the focus moving there, can't hear the voice over in the video but it works as expected on mobile device.

https://drive.google.com/file/d/1cFM95Mf408qi7Oa25YB7R-JWVbXRdyXo/view?usp=sharing

2) **Legal-Standards-Dominican-Republic.mov** -  In this video I have focussed on the Legal Standards section of Dominican Republic and tried to re-create the issue mentioned above, but it works as expected. The VO focus goes beyond 'YES' for item 'State Compulsory Military Recruitment' till the bottom Footnote section and reads it.

https://drive.google.com/file/d/1SSeTce1DOgmiAfOKDsFBcX0AwkZywQGm/view?usp=sharing

These are the only 2 videos I created but I have tested all the scenarios mentioned above and everything is working as expected on iPhone -11 pro and Device simulator.

Please re-test these scenarios and let me know if you still come across these issues.

P.S - You might have to request access for videos since I couldn't find an option for un-restricted access outside of ZTI group.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

It looks like I need to request access to these files. I tried, and it seems I need to use a Gmail account, and I do not have a government-related Gmail account, only a personal one.

It might be easier to test when I get a chance.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

Yes I wasn't able to provide a blanket access to accounts outside of ZTI. I don't it should be a problem if you request access with your personal gmail and view the videos on work computer.

Or if you have a DOL collaboration space I could upload them there. But then you would have to provide access to me for that space.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

Thanks, I plan to retest this issue with VoiceOver today or at least tomorrow and see what happens with my testing again.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

After randomly looking at about a dozen or more countries "Legal Standards" pages, there does not seem to be any more special characters like previously that denoted different things anymore. Also, the test in the footer is being read.

So I'm going to close this issue, and if I find will, I will re-open and test it.
